README.md

```diff
@@ -137,3 +137,28 @@ In particular this means that this project may not depend on GPL-licensed or AGP
 as these would violate the terms of those libraries' licenses.
 
 The contents of this repository are protected by Crown Copyright (C).
+
+## Performance
```

Comment:
I like this section, but I think we should include a bit more information about what was included in each "trasnfer".